### PR TITLE
Migrating to docker compose

### PR DESCRIPTION
**Pull request by gubjanos**
## Release notes

( ) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: plugin/trino-loki/src/test/resources/compose.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+version: '3.4'
+services:
+    loki:
+        image: grafana/loki
```
Comment (jeschkies) [inline plugin/trino-loki/src/test/resources/compose.yaml:4]: Let's add a version.

File: plugin/trino-loki/src/test/resources/compose.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    log-gen:
+        logging:
+          <<: *logging
+        image: mingrammer/flog
+        command: ["-f", "json", "-l", "-d", "2s"]
+        depends_on:
+          - loki
+
+    log-gen-2:
```
Comment (jeschkies) [inline plugin/trino-loki/src/test/resources/compose.yaml:22]: I'd only us one.

File: plugin/trino-loki/src/test/resources/compose.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        ports:
+            - 9000:9000
+            - 9001:9001
+        volumes:
+            - .data-minio:/data:delegated
+
+    grafana:
+        logging:
+            <<: *logging
+        image: grafana/grafana
```
Comment (jeschkies) [inline plugin/trino-loki/src/test/resources/compose.yaml:52]: Same here. We should add a verison.

File: plugin/trino-loki/src/test/resources/docker.compose [hunk 4]
```diff
@@ -0,0 +1,11 @@
+version: '3.4'
+services:
+    loki:
+        image: grafana/loki
+        logging: &logging
+          driver: loki-compose
+          options:
+            loki-url: "http://localhost:8001/loki/api/v1/push"
+            loki-retries: "1"
+            loki-tenant-id: "1"
```
Comment (jeschkies) [inline plugin/trino-loki/src/test/resources/docker.compose:10]: Where is this used?

Comment (gubjanos) [reply plugin/trino-loki/src/test/resources/docker.compose:10]: down in log-gen

Comment (gubjanos) [reply plugin/trino-loki/src/test/resources/docker.compose:10]: and basicly everywhere. Everything is logging here.

File: plugin/trino-loki/src/test/resources/compose.yaml (hunk 1 above)
Comment (gubjanos) [reply plugin/trino-loki/src/test/resources/compose.yaml:4]: good idea!

**Review by jeschkies — APPROVED** [review]

**Pull request merged**